sdks/java/extensions/sql/src/main/java/org/apache/beam/sdk/extensions/sql/impl/rel/BeamCalcRel.java

```diff
@@ -68,15 +121,90 @@ public Calc copy(RelTraitSet traitSet, RelNode input, RexProgram program) {
           BeamCalcRel.class.getSimpleName(),
           pinput);
       PCollection<Row> upstream = pinput.get(0);
+      Schema outputSchema = CalciteUtils.toSchema(getRowType());
+
+      final SqlConformance conformance = SqlConformanceEnum.MYSQL_5;
+      final JavaTypeFactory typeFactory =
```

Comment:
I think `conformance` and typeFactory should be moved out into a separate class similar to `BeamRelDataTypeSystem`